packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -821,6 +836,49 @@ class Auth {
         return token
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizePasswordless.
+     */
+    async authorizePasswordless(parameters: AuthorizePasswordlessParams) {
+        const userid = parameters.userid
+        const mode = this.passwordlessConfig.mode === 'sms' ? 'sms' : 'callback'
+        const callbackURI = this.passwordlessConfig.callbackURI
+
+        await helpers.authorizePasswordless(
+            this.client,
+            {
+                clientSecret: this.clientSecret
+            },
+            {
+                ...(callbackURI && {callbackURI: callbackURI}),
```

Comment:
This condition is always going to be true as we are providing a default value in the constructor. 

What does the SLAS api do if you don't provide a callback?